cwms-radar-client/src/test/resources/radar/v1/json/ts_groups.json

```diff
@@ -1,19 +1,33 @@
 [
+  {
+    "id": "Radar Test",
+    "time-series-category": {
+      "office-id": "SWT",
```

Comment:
when are you going to introduce cwms owned categories?

Reply:
I'm not sure what this means